README.md

```diff
@@ -0,0 +1,60 @@
+New York Times Article Clone
```

Comment:
## Improve the README
- [ ] Inline with having a descriptive README, do you mind adding an alive link to where the page is hosted live? 

You can use this [template ](https://github.com/microverseinc/readme-template)from [pathwright ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)for inspiration. You are also welcomed to use this [tool ](https://raw.githack.com/)to generate a live link.

Reply:
### Improve the Commit and Pull Request Message
- [ ] Your commit messages don't appear to be well informed of what the changes getting added are dong. Do you mind being more elaborate with the commit messages?
- [ ] Kindly note that the Pull Request message ought to communicate what the changes in the said branch are introducing in the main codebase (Changes to be merged). So your mind improving the Pull Request message to showcase to the reviewer what changes/work was carried out in this branch? 

Check this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/) for inspiration on how to go about it.